Add vertical and horizontal scrollbars in collectionview docs. (#982)
telerik/maui-docs: controls/entry/text-appearance.md

> ## Password functionality

Reviewer: ```suggestion
## Password Functionality
```

> The Telerik .NET MAUI Entry provides `IsPassword` (`bool`) property, which when set to `True`, replaces the input with password hint character. The default value is `False`.

Reviewer: ```suggestion
The Telerik .NET MAUI Entry provides the `IsPassword` (`bool`) property. When set to `True`, it replaces the input with a password hint character. The default value is `False`.
```

## Setting the Read-Only State

> You can choose whether the Entry control is editable for end users or not by setting its read-only state through the `IsReadOnly` (`bool`) property. By default, `IsReadOnly` is set to `False`. To restrict the control from editing, set the `IsReadOnly` to `True`.

Reviewer: ```suggestion
You can choose whether the Entry control is editable for end users by setting its read-only state through the `IsReadOnly` (`bool`) property. By default, `IsReadOnly` is set to `False`. To restrict the control from editing, set the `IsReadOnly` to `True`.
```

> ## Setting spacing between the characters

Reviewer: Use Title Case capitalization.

> The `CharacterSpacing` (`double`)property allows you to define the space between the characters inside the control.

Reviewer: ```suggestion
The `CharacterSpacing` (`double`) property allows you to define the space between the characters inside the control.
```

## Setting Text Prediction

> * `IsTextPredictionEnabled` (`bool`)property allows you to define whether text prediction and automatic text correction is on or off. The default value is false. 

Reviewer: ```suggestion
* `IsTextPredictionEnabled` (`bool`) property allows you to define whether text prediction and automatic text correction are on or off. The default value is `False`. 
```

## Defining the Clear Button visibility

> The `ClearButtonVisibility` (`Microsoft.Maui.ClearButtonVisibility`) indicates when the clear-button (the button that clears the text when pressed) can be displayed. The default value is `WhileEditing`. 

Reviewer: ```suggestion
The `ClearButtonVisibility` (`Microsoft.Maui.ClearButtonVisibility`) indicates when the clear button (the button that clears the text when pressed) can be displayed. The default value is `WhileEditing`. 
```
